package.json

```diff
@@ -35,6 +35,7 @@
     "codemirror": "^5.17.0",
     "firebase": "^3.2.1",
     "firepad": "^1.4.0",
+    "prettier": "^1.14.3",
```

Comment:
Can you add a lint script to utilize prettier? Right now prettier is installed but doesn't look like its being used anywhere